documentation/modules/adding-hook-using-ui.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="adding-migration-hook-via-ui_{context}"]
+= Adding a migration hook to a migration plan using the {ocp} web console
+
+You can add a migration hook to an existing migration plan using the {ocp} web console. Note that you need to run one command in the {project-first} CLI.
+
+.Prerequisites
+
+* Migration plan
+* Migration hook file, whose contents you copy and paste into the web console
+
+[NOTE]
+====
+You can run one pre-migration hook, one post-migration hook, or one of each per migration plan.
+====
+
+* File containing the `Secret` for the source provider
+* {ocp} service account that has at least write access for the namespace you are working in
+* SSH access for VMs you want to migrate with the public key installed on the VMs
+* VMs running on Microsoft Server only: Remote Execution enabled
+
+
+.Procedure
+
+. In the {ocp} web console, click *Migration* > *Plans for virtualization* and then click the migration plan you want to add the hook to.
+. Click *Hooks*.
+. For a pre-migration hook, perform the following steps:
+
+.. In the *Pre migration hook* section, toggle the *Enable hook* switch to *Enable pre migration hook*.
+.. Enter the *Hook runner image*. The default image is `quay.io/konveyor/hook-runner` and this value must be used if you are adding an Ansible playbook.
+.. Paste your hook as a YAML file in the *Ansible playbook* text box.
+
+. For a post-migration hook, perform the following steps:
+
+.. In the *Post migration hook*, toggle the *Enable hook* switch *Enable post migration hook*..
+.. Enter the *Hook runner image*. The default image is `quay.io/konveyor/hook-runner` and this value must be used if you are adding an Ansible playbook.
+.. Paste your hook as a YAML file in the *Ansible playbook* text box.
+
+. At the top of the tab, click *Update hooks*.
+. In a terminal, enter the following command to associate each hook with your {ocp} service account:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc -n openshift-mtv patch hook <name_of_hook> \
+    -p '{"spec":{"serviceAccount":"<service_account>"}}' --type merge
+----
+
```

Comment:
@mnecas @matthewsecaur notes:
Issue: The command is correct, but we don't go into any detail about how to create the serviceAccount. It is mentioned in the Prerequisites, so I suppose that is sufficient, but should there be some additional text? Perhaps instead of "associate each hook with your Red Hat OpenShift service account", we could say "associate each hook with the Red Hat OpenShift service account that was created for running hooks" or similar?